src/pages/wines/[wineid].tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+
+import Star from '@/assets/icons/star.svg';
+import { ImageCard } from '@/components/common/card/ImageCard';
+import { ReviewCard } from '@/components/common/card/ReviewCard';
+import UserDefaultImg from '@/components/common/UserDefaultImg';
+import FlavorSliderList from '@/components/wineDetail/FlavorSliderList';
+import Kebab from '@/components/wineDetail/Kebab';
+import LikeButton from '@/components/wineDetail/LikeButton';
+import { testReviews, wineInfo } from '@/components/wineDetail/mock';
+import WineContent from '@/components/wineDetail/WineContent';
+import WineRating from '@/components/wineDetail/WineRating';
+
+export default function WineInfoById() {
+  const { name, region, price, image } = wineInfo;
+
+  return (
+    <main className='mx-auto px-4 md:px-5 xl:px-0 max-w-[1140px]  min-w-[343px]'>
+      <ImageCard
+        imageSrc={image}
+        imageClassName={IMAGE_CLASS_NAME}
+        className='mx-auto relative w-full h-[190px] md:h-[260px] border rounded-[16px] mt-[29px] md:mt-[62px] mb-[40px] md:mb-[60px]'
+      >
+        <WineContent name={name} region={region} price={price} />
+      </ImageCard>
+      <div className='flex flex-col xl:flex-row max-w-[1140px] w-full mx-auto justify-between '>
+        <div className='flex-col  order-2 xl:order-1 xl:max-w-[1140px] '>
+          <h2 className='hidden xl:block mb-[22px] xl:custom-text-xl-bold'>리뷰 목록</h2>
```

Comment:
hidden(display:none)은 시각적으로나 접근성측면에서나 완전 무시되기때문에 특정한 경우가 아니면, 특히 지금과 같은 경우에는 `sr-only`를 추천드려요. (특정한 경우* 탭ui, 모달ui 등)
```suggestion
          <h2 className='sr-only xl:block mb-[22px] xl:custom-text-xl-bold'>리뷰 목록</h2>
```